src/test/java/io/strimzi/kafka/quotas/StaticQuotaCallbackTest.java

```diff
@@ -30,10 +24,27 @@
 import org.junit.jupiter.api.Test;
 import org.mockito.ArgumentCaptor;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I don't think there is consistency in the project about where static imports go, but this is at least consistent with `StaticQutoaConfig` and `StaticQutoaCallback`.